src/ucp/api/ucp.h

```diff
@@ -1198,8 +1197,6 @@ ucs_status_t ucp_get_nbi(ucp_ep_h ep, void *buffer, size_t length,
  * @a rkey. The @a add value is the value that is used for the add operation.
  * When the operation completes the sum of the original remote value and the
  * operand value (@a add) is stored in remote memory.
- * The call to the routine returns immediately, independent of operation
- * completion.
```

Comment:
but does return immediately ? if we can't trigger the operation, we put it on the pending queue, right ?